src/odemis/gui/util/test/img_test.py

```diff
@@ -1048,11 +1048,20 @@ def test_fake_canvas(self):
                         msg="Canvas are equal, which means there in no drawn ruler to be shown in the export")
 
 class TestSpatialExportPyramidal(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # expected max number of pixels with a safety margin, derived from the MAX_RES_FACTOR in img.py
+        cls.expected_max_nmr_pixels = 23004160 * 0.95
 
-    def setUp(self):
```

Comment:
Please leave the `setUp()` with the `wx.App()` inside it, and move the rest to a function more clearly named. Maybe something like `_prepare_streams(fluo_res, sem_res)`.
Maybe that function could just return the streams, instead of changing .streams, but as it might involve more changes to the original test cases, let's not do that now.

Reply:
indeed returning will be to much work, therefore I will implement the first suggested method.